### PR TITLE
Protected members

### PR DESCRIPTION
**Pull request by AssafTzurEl**
Make OmsOutput  fields healthReporter and connectionData protected, in order to allow access by OmsOutput subclasses.

### CODE REVIEW

File: src/Microsoft.Diagnostics.EventFlow.Outputs.Oms/OmsOutput.cs [hunk 1]
```diff
@@ -3,6 +3,11 @@
 //  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
 // ------------------------------------------------------------
 
+using Microsoft.Diagnostics.EventFlow.Configuration;
```
Comment (karolz-ms) [inline src/Microsoft.Diagnostics.EventFlow.Outputs.Oms/OmsOutput.cs]: It does not really matter that much but we have a slight preference for keeping System namespaces above others. There is a setting in Visual Studio that helps maintain that order.

**Comment by karolz-ms** [comment]
@AssafTzurEl please let me know if you need a new version of OmsOutput (with this change) to be published on Nuget

File: src/Microsoft.Diagnostics.EventFlow.Outputs.Oms/OmsOutput.cs (hunk 1 above)
Comment (AssafTzurEl) [reply src/Microsoft.Diagnostics.EventFlow.Outputs.Oms/OmsOutput.cs]: @karolz-ms  Thanks. I'm using the "remove and sort" command in VS, and I like it - the sorting helps me locate using's faster.

**Comment by AssafTzurEl** [comment]
@karolz-ms re/ new Nuget version: No rush.

**Pull request merged**